inst/automation/pull_ropensci-archive2.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
maybe instead of duplicating scripts I could learn how to parameterize them... do you know how to do that? :-)

Reply:
not sure how to do that